doc/neomake.txt

```diff
@@ -41,6 +41,11 @@ fantastic vim plugins such as syntastic and dispatch.
                         used. If no default top-level makers exist, |makeprg|
                         will be used.
 
+                                                            *neomake-:NeomakeSh*
+:NeomakeSh {command}    Run {command} in a shell (according to 'shell'). The
+                        command output will be loaded in the quickfix list when
+                        the job is complete.
```

Comment:
Can you add an example at the end? Something like `:NeomakeSh find . -name '*.pyc'`.